mimic/rest/loadbalancer_api.py

```diff
@@ -26,20 +27,20 @@
 @implementer(IAPIMock, IPlugin)
 class LoadBalancerApi(object):
     """
-    Rest endpoints for mocked Load balancer api.
+    This class registers the load balancer API in the service catalog.
+    It does NOT implement individual endpoints.
```

Comment:
Because the original docstring (originating from nova's code) says that it implements endpoints.  I spent 4 days wondering why nothing worked.